test/contrib/geneve.uts

```diff
@@ -7,7 +7,8 @@
 
 = Build & dissect - GENEVE encapsulates Ether
 if WINDOWS:
```

Comment:
Could we add these lines to UTScapy instead?

Reply:
Done